tee-worker/omni-executor/rpc-server/src/methods/omni/common.rs

```diff
@@ -63,6 +70,27 @@ impl From<PumpxRpcError> for ErrorObjectOwned {
 	}
 }
 
+impl From<DetailedError> for PumpxRpcError {
+	fn from(error: DetailedError) -> Self {
+		Self {
+			code: error.code,
+			message: error.message,
+			data: Some(PumpxRpcErrorData {
+				backend_response: Some(PumpxRpcErrorBackendResponse {
+					code: error.code,
```

Comment:
Why `backend_response` is always present ?
It means it will be also present for errors occured in omni-executor and might be not related to pumpx backend - am I right ?

Reply:
`backend_response` should be `None` by default, I missed that. It will only be present when is `Some`. I'll fix it